### PR TITLE
ICU-20857 update API Change Report for ICU 66preview

### PR DESCRIPTION
**Pull request by srl295**
[ICU-20857] update API Change Report for ICU 66preview

[ICU-20857]: https://unicode-org.atlassian.net/browse/ICU-20857

### CODE REVIEW

File: icu4c/APIChangeReport.md [hunk 1]
```diff
@@ -1,642 +1,92 @@
 
-  
 <!--
- Copyright © 2019 and later: Unicode, Inc. and others.
+ Copyright © 2019: Unicode, Inc. and others.
  License & terms of use: http://www.unicode.org/copyright.html
 -->
 
-# ICU4C API Comparison: ICU 64 (update #1: 64.2) with ICU 65
+# ICU4C API Comparison: ICU 65 with ICU 66 (m1)
```
Comment (markusicu) [inline icu4c/APIChangeReport.md]: preview, not m1

File: icu4c/APIChangeReport.md [hunk 2]
```diff
@@ -1,642 +1,92 @@
 
-  
 <!--
- Copyright © 2019 and later: Unicode, Inc. and others.
+ Copyright © 2019: Unicode, Inc. and others.
```
Comment (markusicu) [inline icu4c/APIChangeReport.md]: we should preserve the "and later" part

File: icu4c/APIChangeReport.html [hunk 3]
```diff
@@ -38,2951 +38,1455 @@ <h1>ICU4C API Comparison: ICU 64 (update #1: 64.2) with ICU 65</h1>
...
 <THEAD>
 <tr>
-<th>File</th><th>API</th><th>ICU 64</th><th>ICU 65</th>
+<th>File</th><th>API</th><th>ICU 65</th><th>ICU 66</th>
 </tr>
 </THEAD>
 <tr class="row1">
-<td class="file">decimfmt.h</td><td class="proto">int32_t icu::DecimalFormat::getMultiplierScale() const</td><td class="" colspan="2" align="center">Draft&rarr;Stable<br>ICU 62</td>
+<td class="file">uchar.h</td><td class="proto"><tt>enum</tt> UBlockCode::UBLOCK_CHORASMIAN</td><td class="">(missing)</td><td>Stable<br>
+<span class=""><span>ICU 66</span></span></td><td class="bornstable"><b class="bigwarn" title="A new API was introduced as stable in $rightVer.">(Born Stable)</b></td>
```
Comment (markusicu) [inline icu4c/APIChangeReport.html]: In the title attribute, `$rightVer` is not getting replaced.

File: icu4c/APIChangeReport.md (hunk 1 above)
Comment (srl295) [reply icu4c/APIChangeReport.md]: This is auto generated based on the version number, but I can manually change it?

File: icu4c/APIChangeReport.html (hunk 3 above)
Comment (srl295) [reply icu4c/APIChangeReport.html]: fixed

**Pull request merged**